ps_linklist.php

```diff
@@ -106,7 +112,15 @@ public function install()
             return false;
         }
 
-        $installed = $this->installFixtures();
+        $installed = $this->createTables();
```

Comment:
Could we rename a bit these variables 😅  ? They are used for so many purposes, the code is hard to follow.

I suggest
```
    public function install()
    {
        if (!parent::install()) {
            return false;
        }

        $tablesInstalledWithSuccess = $this->createTables();

        if (!tablesInstalledWithSuccess) {
            $this->uninstall();
            return false;
        }


        $old16ModuleUninstalledWithSuccess = $this->uninstallPrestaShop16Module();

        if (old16ModuleUninstalledWithSuccess) {
            (new DataMigration(Db::getInstance()))->migrateData();
            $dataLoadedWithSuccess = true;
        } else {
            $dataLoadedWithSuccess = $this->installFixtures();
        }
        
        if ($dataLoadedWithSuccess 
            && $this->registerHook('displayFooter')
            && $this->registerHook('actionUpdateLangAfter')) {
            return true;
        }

        $this->uninstall();

        return false;
    }
```

I used overly explicit variable names on purpose but you can choose something you prefer more 😉